bench.py

```diff
@@ -484,7 +486,7 @@ def print_output(library, algorithm, stages, params, functions,
         output = []
         for i in range(len(stages)):
             result = gen_basic_dict(library, algorithm, stages[i], params,
-                                    data[i], alg_instance, alg_params)
+                                    data[i], alg_instance, alg_params if i == 0 else None)
```

Comment:
Not clear why only first stage has `alg_params`

Reply:
It seems like in all benchmarks all stages in each case has similar parameters. So, since the the parameter list is usually quite long, we can reduce the length of benchmark output by printing this section only once.

Reply:
Excel report generator filters benchmark cases basing on parameters, output should not be shortened for correct work of generator

Reply:
rolled back that change, but very upset about it :(